server/controllers/recipesController.js

```diff
@@ -1,25 +1,31 @@
-const axios = require('axios');
+const axios = require("axios");
+const jwt = require("jsonwebtoken");
 
 /* Start n8n workflow using ingredients list provided*/
 // POST /api/recipes/suggest
 const suggestRecipes = async (req, res) => {
-    const { ingredients } = req.body; // Receive ingredients list from the front-end
+  const { prompt } = req.body; // Receive ingredients list from the front-end
 
-    try {
-        // URL of N8n webhook per https://docs.n8n.io/integrations/builtin/core-nodes/n8n-nodes-base.webhook/?utm_source=n8n_app&utm_medium=node_settings_modal-credential_link&utm_campaign=n8n-nodes-base.webhook
-        const webhookUrl = `${process.env.N8N_WEBHOOK_URL}/recommend-recipes`;
-        const response = await axios.post(webhookUrl, { ingredients });
+  try {
+    // URL of N8n webhook per https://docs.n8n.io/integrations/builtin/core-nodes/n8n-nodes-base.webhook/?utm_source=n8n_app&utm_medium=node_settings_modal-credential_link&utm_campaign=n8n-nodes-base.webhook
+    //const webhookUrl = `${process.env.N8N_WEBHOOK_URL}/recommend-recipes`;
+    const webhookUrl = process.env.N8N_WEBHOOK_URL_RECIP;
```

Comment:
See my comment to Jacob https://github.com/nicmart-dev/feedmenow/pull/36#discussion_r1697715660

Reply:
The .env has been updated and proper values placed.